gammapy/estimators/flux.py

```diff
@@ -88,9 +88,15 @@ def _set_norm_parameter(self, norm=None, scaled_parameter=None):
         norm.value = 1.0
         norm.frozen = False
 
-        norm.min = scaled_parameter.min / scaled_parameter.value
-        norm.max = scaled_parameter.max / scaled_parameter.value
-        norm.interp = scaled_parameter.interp
+        if scaled_parameter:
+            norm.min = scaled_parameter.min / scaled_parameter.value
+            norm.max = scaled_parameter.max / scaled_parameter.value
+            norm.interp = scaled_parameter.interp
+        else:
+            norm.min = self.norm_min
+            norm.max = self.norm_max
```

Comment:
the FluxEstimator defaults for norm_mina and norm_max are 0.2, and 5 respectively, so the constraint may be too strong if the reference model is not optimal then NaN default of the parameter may be better.